README.md

```diff
@@ -2019,6 +2019,7 @@ Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL_
 - [Clink](https://short.swurl.xyz/) - Dead-simple, super minimal URL shortener in C. ([Source Code](https://git.swurl.xyz/swirl/clink)) `AGPL-3.0` `C`
 - [Easyshortener](https://github.com/easypanel-community/easyshortener) - A simple URL shortener. `MIT` `PHP/Nodejs/Docker`
 - [goshorly](https://gitea.hackmi.ch/Phil/goshorly) - An easy self-hosted Link shortener in Golang. `MIT` `Go`
+- [Just Short It!](https://github.com/miawinter98/just-short-it) - A KISS, single-user URL shortener that runs in just one container. `MIT` `Docker`
```

Comment:
Where can we find the source code for `mcr.microsoft.com/dotnet/aspnet:7.0` and `r.microsoft.com/dotnet/sdk:7.0` used in https://github.com/miawinter98/just-short-it/blob/main/Dockerfile?

Reply:
https://github.com/dotnet/dotnet-docker